### ISSUE TITLE
Custom Widget in Element Web crashes on some browsers

### ISSUE THREAD

**Issue by stephanerosa** [vector-im/element-web#15513]
### Description

I have tried to embed all kinds of website in a Custom Widget, including something as basic as http://neverssl.com but although the widget is created successfully, it crashes upon click.

### Steps to reproduce

- In a room, add a custom widget from vector.im
- In the URL, set http://neverssl.com, Save
- Click the Custom widget -> crash.

What I expect is to be able to have a sort of iframe in the panel, with the goal of displaying a user guide for the users that I will maintain on another webserver of mine.

Log: sent/not sent?    

### Version information

- **Platform**: On Element-web, it crashes. On Element-Android, it works perfect, on Element-Desktop it works perfect

For the web app:

- **Browser**: Waterfox Classic 2020-09 : Crash. On Edge Chromium 86.0.622.43 it works perfect
- **OS**: Windows 10
- **URL**: Private Server with Cloudron maintained versions of Matrix and Element.io (always latest stable)

For the desktop app:

- **OS**: Windows 10
- **Version**: 1.7.10

**Comment by t3chguy** [comment]
`2020-10-20T22:23:32.959Z E Illegal constructor.`

The error is coming from the widget itself

```
2020-10-20T22:23:32.994Z E The above error occured while React was rendering the following components: 
    in iframe
```

**Comment by t3chguy** [comment]
This bug is fixed in 1.7.10 - please update.

**Comment by stephanerosa** [comment]
Ironically, it self-updated as expected only a couple hours after my post :-)

```
App Info
App Title & Version            Element 1.7.10
App ID                         e4a812d0-07cc-4b3e-a39d-dda7feb45fe2
Package Version                v1.4.8
Last Updated                   4 hours ago
```
Works perfect now, even on unsupported Waterfox. Sorry for the bad timing reporting this ;-)
